src/classes/STG_InstallScript_TEST.cls

```diff
@@ -244,4 +244,60 @@ public with sharing class STG_InstallScript_TEST {
             System.assertNotEquals(null, addr.Parent_Contact__c);
         }
     }
+
+    @isTest
+    public static void createAdminAccountForContactOnInstall() {
+
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe.getAdminAccRecTypeID()));
+
+        Contact cont = new Contact(
+                FirstName = 'Test',
+                LastName = 'User',
+                Email = 'usertest@domain.com'
+        );
+
+        TDTM_ProcessControl.setRecursionFlag(TDTM_ProcessControl.registeredTrigger.ACCT_IndividualAccounts_TDTM_After_Insert, true);// need to turn off the trigger for this test
```

Comment:
The indentation looks a little weird to me here. Also, I am not a big fan of putting comment after a line of code, just my opinion.